packages/divider-block/src/settings.ts

```diff
@@ -200,15 +200,15 @@ export const settings: BlockSettings = {
                     choices: [
                         {
                             value: DividerStyle.Solid,
-                            label: 'Solid',
+                            label: DividerStyle.Solid,
```

Comment:
DividerStyle represent the values not the labels, it is pure luck that they match (if we add word with space then we will have a weird PascalCase label), do you really want to have this change?

Reply:
Fair point, changed the label to a string :)